### ISSUE TITLE
Menu is swipeable while dialog is open on mobile

### ISSUE THREAD

**Issue by TerrellLinnell** [codexstanford/techlist-frontend-web#105]
While accessing create company form from the nav on mobile, if you try to scroll the dialog, it scrolls the menu in the background instead.

**Comment by medelman17** [comment]
@TerrellLinnell -- is this still an issue?

**Comment by TerrellLinnell** [comment]
we currently just close the menu when we open the dialog on mobile to prevent the action being intercepted. It works fine so this seems like something we can wait on for now unless it's important that when a user gets to the create company form from the menu, then closes the create company dialog, that they be shown the menu which they left from to get here.  It's essentially one extra tap for the user right now to open the menu again after opening the create company form from the menu on mobile, could be nice to have it work with the menu open but not something I'd say is a priority.
